db/migrate/20170619185947_add_timestamps_to_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimestampsToActivities < ActiveRecord::Migration[5.0]
+  def change
+    add_timestamps(:activities)
```

Comment:
When your actively working on a feature and have a migration that hasn't been merged into master, you can always do a `rake db:rollback` and edit your migration to add in things you missed like `timestamps`. No need to make a chance here, just a heads up for the future. Once you've merged a migration into master though, you don't want to edit the migration again.